lib/fetch/helpers.js

```diff
@@ -137,5 +137,13 @@ export const serializeData = (data, input) => {
             body: serializeFormData(data)
         };
     }
+    if (input === 'binary') {
```

Comment:
This part will not be needed though if/after https://gitlab.protontech.ch/proton/fe/proton-drive/merge_requests/20 is merged